src/sinks/aws_cloudwatch_logs/mod.rs

```diff
@@ -755,6 +755,54 @@ mod integration_tests {
         assert_eq!(output_lines, input_lines);
     }
 
+    #[test]
+    fn cloudwatch_dynamic_group_and_stream_creation() {
+        let mut rt = Runtime::new().unwrap();
+
+        let stream_name = gen_stream();
+
+        let region = Region::Custom {
+            name: "localstack".into(),
+            endpoint: "http://localhost:6000".into(),
+        };
+
+        let config = CloudwatchLogsSinkConfig {
+            stream_name: stream_name.clone().into(),
+            group_name: GROUP_NAME.into(),
```

Comment:
How do we know this group doesn't already exist?

Reply:
good catch fixed in https://github.com/timberio/vector/pull/759/commits/4d71468c85db45fa8ae2d2d6f1e320a22708d3a0